content/influxdb/cloud-dedicated/primers/api/v1/_index.md

```diff
@@ -0,0 +1,275 @@
+
+The following sample shows how to configure the `outputs.influxdb` Telegraf plugin for InfluxDB Cloud Dedicated:
+
+```toml
+[[outputs.influxdb]]
+  urls = ["http://localhost:8086"]
+  database = "DATABASE_NAME"
+  skip_database_creation = true
+  retention_policy = ""
+  username = ""
+  password = "DATABASE_TOKEN"
+  content_encoding = "gzip”
+```
+
+Replace the following:
+
+- **`DATABASE_NAME`**: your InfluxDB Cloud Dedicated database
+- **`DATABASE_TOKEN`**: a [Database Token](/influxdb/cloud-dedicated/get-started/setup/#create-a-database-token) with permission to write to the database
+
+#### Other Telegraf configuration options
+
+`influx_uint_support`: supported in InfluxDB IOx.
+
+### Write data using v1 client libraries
+
+### Write data using v1 API /write
+
+Use HTTP clients and your custom code to send write requests to the v1 API `/write` endpoint.
+
+{{% api-endpoint endpoint="http://localhost:8086/write" method="post"%}}
+
+#### v1 API /write parameters
+
+Parameter     | Allowed in   | Ignored | Value
+--------------|--------------|---------|------------------------------------------------------------
+`consistency` | Query string | Ignored | N/A
+`db`          | Query string | Honored | Database name
+`precision`   | Query string | Honored | [Timestamp precision](#timestamp-precision): `ns`, `u`, `ms`, `s`, `m`, `h`
+`rp`          | Query string | Honored | Honored, but discouraged
+`u`           | Query string | Ignored | Any string or empty
+`p`           | Query string | Honored | [Database token]() with permission to write to the database
+`Content-Encoding` | Header      | Honored | `gzip` (compressed) or `identity` (uncompressed)
+`Authorization`    | Header      | Honored | `Token DATABASE_TOKEN` | `Basic <base64 [username]:DATABASE_TOKEN>`
+
+#### Timestamp precision
+
+Use one of the following `precision` values in v1 API `/write` requests:
+
+- `ns`: nanoseconds
+- `u`: microseconds <!-- @TODO: test that differs from `us` used in v2?? -->
+- `ms`: milliseconds
+- `s`: seconds
+- `m`: minutes
+- `h`: hours
+
+### Write data using HTTP clients
+
+To test interactively, use common HTTP clients such as cURL and Postman to send requests to the v1 API `/write` endpoint.
+
+{{% warn %}}
+While the v1 CLI may coincidentally work with {{% cloud-name %}}, it isn't supported.
+{{% /warn %}}
+
+The following example shows to use the **cURL** command line tool to write line protocol data to an InfluxDB Cloud database:
+
+```sh
+curl -i http://localhost:8086/write?db=DATABASE_NAME&precision=s \
+    --header 'Authorization: Token DATABASE_TOKEN' \
+    --data-binary 'home,room=kitchen temp=72 1463683075'
+```
+
+### v1 CLI (not supported)
+
+Don't use the v1 CLI for writing data to {{% cloud-name %}}.
+While the v1 CLI may coincidentally work with {{% cloud-name %}}, it isn't supported.
+
+If you need to test writes interactively, see how to [write data using HTTP clients](#write-data-using-http-clients).
```

Comment:
The v1 `influx write` command should actually work. It just builds line protocol and sends it to the `/write` endpoint. What probably won't work is the InfluxQL interactive shell built into the CLI.

Reply:
@sanderson OK, but I understand the Dedicated UX doc to say _avoid_ it:

> if a user wishes to test writes interactively, especially for testing, they would likely use the v1 CLI to do so. While this may coincidentally work, for April 26th we will make no special efforts to support it. In such cases, users can use CURL.